samples/CoreAnimationTest/CoreAnimationTest/CALayerMultipleViewController.m

```diff
@@ -22,23 +22,24 @@
 #import "SwitchTableViewCell.h"
 #import "CGPointTableViewCell.h"
 #import "CGRectTableViewCell.h"
-#import "ViewHeaderFooterView.h"
+#import "HeaderTableViewCell.h"
+#import "FooterTableViewCell.h"
 
 #define VC_WIDTH self.view.frame.size.width
 #define VC_HEIGHT self.view.frame.size.height
 
-#define HIDDEN_POS [_exposedProperties.allKeys indexOfObject:kHiddenProperty]
-#define OPACITY_POS [_exposedProperties.allKeys indexOfObject:kOpacityProperty]
-#define CONTENTS_GRAVITY_POS [_exposedProperties.allKeys indexOfObject:kContentsGravityProperty]
-#define CONTENTS_POS [_exposedProperties.allKeys indexOfObject:kContentsProperty]
-#define SHOULD_RASTERIZE_POS [_exposedProperties.allKeys indexOfObject:kShouldRasterizeProperty]
-#define RASTERIZATION_SCALE_POS [_exposedProperties.allKeys indexOfObject:kRasterizationScaleProperty]
-#define CONTENTS_SCALE_POS [_exposedProperties.allKeys indexOfObject:kContentsScaleProperty]
-#define DOUBLE_SIDED_POS [_exposedProperties.allKeys indexOfObject:kDoubleSidedProperty]
-#define GEOMETRY_FLIPPED_POS [_exposedProperties.allKeys indexOfObject:kGeometryFlippedProperty]
-#define CORNER_RADIUS_POS [_exposedProperties.allKeys indexOfObject:kCornerRadiusProperty]
-#define BORDER_WIDTH_POS [_exposedProperties.allKeys indexOfObject:kBorderWidthProperty]
-#define MASKS_TO_BOUNDS_POS [_exposedProperties.allKeys indexOfObject:kMasksToBoundsProperty]
+#define HIDDEN_POS [_exposedProperties indexOfObject:kHiddenProperty] + 1
```

Comment:
Headers are now table cells as well.

---
In reply to: [99435804](https://github.com/Microsoft/WinObjC/pull/1874#discussion_r99435804) [](ancestors = 99435804)